test/models/guitar_test.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class GuitarTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
Please add a test or delete the file

Reply:
File removed 👍🏼 
Resolving this conversation